pkg/storage/storage.go

```diff
@@ -77,9 +77,10 @@ type coreStorage struct {
 	Storage
 	regionStorage endpoint.RegionStorage
 
-	useRegionStorage int32
-	regionLoaded     bool
-	mu               syncutil.Mutex
+	useRegionStorage        int32
```

Comment:
Is that safe to switch between using or not using RegionStore in the master branch or the latest releases now?

Reply:
defaultUseRegionStorage is true and only check switch when it was just elected leader.